### DISCUSSION TITLE
Proxy | WenjieWei Blog

### DISCUSSION THREAD

**Discussion by giscus** [wjwei-handsome/wjwei-docusaurus#14] [General]
# docs/Potpourri/Proxy

针对不同场景，灵活科学上网的更佳体验

https://www.wjwei.blog/docs/Potpourri/Proxy

**Comment by PeixiongYuan** [comment]
用小猫咪的bypass功能 添加系统代理的白名单好像不会出现被覆盖的问题

**Reply by wjwei-handsome** [reply]
bypass在windows版本软件中提供了入口修改，在mac上可以手动添加ignoreList
